src/GovUk.Education.ExploreEducationStatistics.Data.Services.Tests/SubjectMetaServiceTests.cs

```diff
@@ -42,33 +48,42 @@ public class SubjectMetaServiceTests
         private readonly LocalAuthority _sunderland = new("E08000024", "", "Sunderland");
 
         [Fact]
-        public async Task GetSubjectMeta_SubjectNotFound()
+        public async Task GetCachedSubjectMeta_SubjectNotFound()
```

Comment:
Are we going with the convention of prefixing cached methods with 'Cached'? This conversation has also cropped up in Duncan's PR: https://github.com/dfe-analytical-services/explore-education-statistics/pull/3277#discussion_r861799083

Personally, I'm not a big fan for the aforementioned reasons in that discussion, but perhaps we need to discuss this a little further.

Reply:
Thanks, I'm not sure. I made the change to prefix the name with 'Cached' after Duncan chatted with me about it and left a comment (see above).

I think we do need to discuss this further as I see this cropping up on 3 PR's now! 🙂